ibu-imager/clusterinfo/clusterinfo.go

```diff
@@ -73,6 +74,11 @@ func (m *InfoClient) CreateClusterInfo(ctx context.Context) (*ClusterInfo, error
 		return nil, err
 	}
 
+	hostname, err := m.getNodeHostname(ctx)
```

Comment:
Thinking maybe better bringing node object once and provide it to getNodeInternalIP and getNodeHostname ?

Reply:
Agreed, I also thought of that when adding this method, but I thought it adds more complexity than saves time. But let me try and refactor this piece, we should be able to do better indeed :)

Reply:
Implemented your idea and it seems cleaner/clearer PTAL.